feature-configs/deploy/gatekeeper/webhook.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: gatekeeper-mutating-webhook-configuration
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
```

Comment:
that's a weird one
why do we want a newline there?

Reply:
good question maybe the deployment updates the CA after that?

@mmirecki can you help here?

Reply:
Gatekeeper will generate one.
Please check [here](https://github.com/open-policy-agent/cert-controller) for more details.